exporter/prometheusexporter/prometheus_test.go

```diff
@@ -92,6 +92,7 @@ func TestPrometheusExporter_endToEnd(t *testing.T) {
 			Name:        "this/one/there(where)",
 			Description: "Extra ones",
 			Unit:        "1",
+			Type:        metricspb.MetricDescriptor_CUMULATIVE_INT64,
```

Comment:
Note one other thing that changed is that untyped metrics by default will show as "untyped" instead of "counter"